File: The Core/C#/circleOfNumbers.cs
int circleOfNumbers(int n, int firstNumber) {
    List<int> numbers = new List<int>();

    for(int i = 0; i < n; i++){
        numbers.Add(i);
    }
    
    if(firstNumber < n/2) return numbers[firstNumber + n/2];    
    else return numbers[firstNumber - n/2];
        
}

File: The Core/Intro Gates/6. Circle Of Numbers/circleOfNumbers.js
function circleOfNumbers(n, firstNumber) {
    var numArr = new Array();

    for(i = 0; i < n; i++){
        numArr.push(i);
    }
    
    if(firstNumber < n/2) return numArr[firstNumber + n/2];    
    else return numArr[firstNumber - n/2];     
}
